protected/humhub/modules/ui/form/widgets/ActiveField.php

```diff
@@ -81,4 +92,145 @@ public function end()
 
         return parent::end();
     }
+
+    /**
+     * @inheritdoc
+     */
+    public function checkbox($options = [], $enclosedByLabel = true)
+    {
+        return parent::checkbox($this->applyRegularSingleStyle($options, [
+            'inputClass' => 'regular-checkbox',
+            'boxClass' => 'regular-checkbox-box',
+            'template' => $this->checkboxTemplate,
+        ]), $enclosedByLabel);
+    }
+
+    /**
+     * @inheritdoc
+     */
+    public function checkboxList($items, $options = [])
+    {
+        return parent::checkboxList($items, $options)->applyRegularListStyle([
+            'type' => 'checkbox',
+            'containerFindClass' => 'checkbox',
+            'containerAdditionalClass' => 'regular-checkbox-container',
+            'inputClass' => 'regular-checkbox',
+            'boxClass' => 'regular-checkbox-box',
+        ]);
+    }
+
+    /**
+     * @inheritdoc
+     */
+    public function radio($options = [], $enclosedByLabel = true)
+    {
+        return parent::radio($this->applyRegularSingleStyle($options, [
+            'inputClass' => 'regular-radio',
+            'boxClass' => 'regular-radio-button',
+            'template' => $this->radioTemplate,
+        ]), $enclosedByLabel);
+    }
+
+    /**
+     * @inheritdoc
+     */
+    public function radioList($items, $options = [])
+    {
+        return parent::radioList($items, $options)->applyRegularListStyle([
+            'type' => 'radio',
+            'containerFindClass' => 'radio',
+            'containerAdditionalClass' => 'regular-radio-container',
+            'inputClass' => 'regular-radio',
+            'boxClass' => 'regular-radio-button',
+        ]);
+    }
+
+    /**
+     * Apply style "regular" for single of checkbox or radio input
+     *
+     * @param array $options
+     * @param array $regularOptions
+     * @return array
+     */
+    protected function applyRegularSingleStyle(array $options, array $regularOptions): array
```

Comment:
Please make `protected` methods `private` when possible.

Reply:
@luke- Commit https://github.com/humhub/humhub/pull/5815/commits/ba0c29a6f585ab365d14bad89f4054c0f20908dc, yes, currently it is possible, but I made them `protected` because I thought if some module will need it for extended class.